etc/httpd/global.conf

```diff
@@ -202,10 +202,8 @@ RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?collusion.*$ /$1lightbeam/ [L,R=301]
 RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?firefox/toolkit/download-to-your-devices(.*)$ /$1firefox/ [L,R=301]
 
 ## Home Page
-# Has to be this way because of old
```

Comment:
Oh yeah, and `/mwc/`. This will indeed just need to be at the bottom of the file so it's a catchall if nothing else matches.